### PR TITLE
Seed database

### PR DESCRIPTION
**Pull request by castaneda-ed**
I seeded the database with 10 events, 100 users and 25 interests. Each event has 10 attending users and each User has 5 interests assigned. I tried my best to comment what the code does and used js documentation for better understanding.  I also pulled new changes and merged the to my files before pushing. I did a migrate reset for testing the database.  I used the better options that I found on faker for the events 😁 did not seed anything for messages table.

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -8,66 +8,57 @@ datasource db {
 }
```
Comment (castaneda-ed) [inline prisma/schema.prisma:9]: I don't know why it shows that I did changes to this file, cause I didn't, but everything is working.

**Review by wangbertha — COMMENTED** [review]
Could we leave the schema.prisma file as-is from our Zoom meeting today? Just since we all worked through it together and are familiar with the naming conventions and all. Was there a reason you wanted to update some of the naming and ordering of the properties?

Seeding looks good to me and thorough!

File: prisma/schema.prisma (hunk 1 above)
Comment (wangbertha) [reply prisma/schema.prisma:9]: Sorry, I just saw this after submitting my first comment! It looks like there are some unintentional naming changes. My guess is, maybe Prettier updated it? I wonder if we can revert the changes.

Comment (castaneda-ed) [reply prisma/schema.prisma:9]: I don't know what happened, but the only thing I did with the schema was to reference, but not changes, maybe I can resubmit my changes after I change the prisma schema back to what it was before?

**Pull request closed without merging**